Makefile

```diff
@@ -148,8 +147,8 @@ dist:
 .PHONY: clean
 clean:
 	go mod tidy
-	rm $(BIN)
-	rm $(DATA)
+	rm -f $(BIN)
```

Comment:
Maybe adding the `-` prefix to these lines so they don't cause `make` to exit if the command exits with a non-zero exit code?

Reply:
Interesting: https://www.gnu.org/software/make/manual/make.html#index-_002d-_0028in-recipes_0029, but it still prints errors about non-existing files to `stdout`, and I think we want to suppress printing errors too.

Reply:
I think you'd have to redirect stderr to /dev/null in that case.

Reply:
I know, but I consider current solution as more straightforward.

Reply:
@subpop Hi, are you requesting your proposal or is my solution good enough?

Reply:
Oh. This is fine with me. I'm not concerned with suppressing output.